### PR TITLE
Rnr 17 subnav landing page

### PR DESCRIPTION
**Pull request by jack829**
Removes the bottom tabbar
Creates subnav for switching between "Wizards" chats and Friends' chats
Leaves floating button to add a new tinder chat

### CODE REVIEW

**Comment by jack829** [comment]
simulator screen shot - iphone xs max - 2018-10-15 at 17 28 20

**Comment by jack829** [comment]
simulator screen shot - iphone xs max - 2018-10-15 at 17 28 20

**Comment by jack829** [comment]
simulator screen shot - iphone xs max - 2018-10-15 at 17 28 04

**Comment by jack829** [comment]
screen shot 2018-10-15 at 5 28 09 pm

File: src/HOC/importNewMatchWrapper.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    onPressAddMatch() {
+      Actions.newChatStack()
+    }
+
+    render() {
+      const buttonPosition = {
+        left: this.state.buttonLeft
+      }
+      console.log('buttonPosition ', buttonPosition)
```
Comment (dblanchard13) [inline src/HOC/importNewMatchWrapper.js]: Think this can safely be removed.

File: src/screens/ChatList.js [hunk 2]
```diff
@@ -315,4 +377,4 @@ const styles = StyleSheet.create({
   },
 })
 
-export default ChatList
+export default importNewMatchWrapper(ChatList)
```
Comment (dblanchard13) [inline src/screens/ChatList.js]: Update name to something along the lines of `addNewScryChatButtonWrapper`

**Pull request merged**